tools/crm_attribute.c

```diff
@@ -312,7 +312,7 @@ main(int argc, char **argv)
 
     GOptionGroup *output_group = NULL;
     pcmk__common_args_t *args = pcmk__new_common_args(SUMMARY);
-    gchar **processed_args = pcmk__cmdline_preproc(argv, "DGNPdilnpstv");
+    gchar **processed_args = pcmk__cmdline_preproc(argv, "NPdilnpstv");
```

Comment:
also U

Reply:
I checked these two or three times, and somehow still missed this and the other one.  Fixed both.